packages/nuejs.org/docs/performance-optimization.md

```diff
@@ -30,15 +30,15 @@ Setting this globally is also possible, but then you need to re-compile all page
 
 ## View transitions
 The next most important performance setting is to enable view transitions.
-This causes your pages to load significantly faster, and the page is transitioned to another in an animated way.
+This causes your pages to load significantly faster, and the page transition in an animated way.
```

Comment:
```suggestion
This causes your pages to load significantly faster and transition in an animated way.
```

Maybe `and to transition`?

Reply:
That reads much better and no need of comma. Thanks.